crowbar_framework/app/models/node_object.rb

```diff
@@ -256,6 +256,27 @@ def intended_role=(value)
     @node["crowbar_wall"]["intended_role"] = value
   end
 
+  def disks_roles=(value)
+    @node["crowbar_wall"] ||= {}
+    @node["crowbar_wall"]["disks_roles"] = value
+  end
+
+  def disks_roles
+    disks_roles = crowbar_wall["disks_roles"] || {}
```

Comment:
I know it's WIP, but just one question: is it intended that you're not using `crowbar_wall["claimed_disks"]`?